nuxt-frontend/components/home-new/LandingSection.vue

```diff
@@ -11,100 +11,99 @@
     />
 
     <div
-      class="container flex flex-col items-center lg:flex-row-reverse lg:items-start pb-[1.563rem] md:pb-[3.125rem]"
+      class="container flex flex-col items-center lg:flex-row-reverse lg:items-start pb-[1.563rem] lg:pb-[3.125rem]"
     >
-      <aspect-ratio class="flex-[45%] lg:block hidden mt-10" height="66%">
+      <aspect-ratio
+        class="flex-[45%] lg:block hidden mt-10 2xl:mt-6"
+        height="66%"
+      >
         <img
           class="w-full h-full object-contain"
           :src="landing800"
           :srcset="`${landing400} 400w, ${landing800} 800w`"
           alt="landing-view-image"
         />
       </aspect-ratio>
-      <div class="flex-[55%] flex flex-col mt-10 md:mt-5 lg:mt-[6.25rem]">
+      <div
+        class="flex-[55%] flex flex-col items-center lg:items-start mt-6 lg:mt-[6.25rem]"
+      >
         <h1
-          class="w-[100%] sm:w-[85%] pb-5 lg:pb-8 xl:pb-10 tracking-[0] text-[2.1875rem] leading-[2.8125rem] md:text-[2.65625rem] md:leading-[3.28125rem] lg:text-[3.28125rem] lg:leading-[3.70625rem] 2xl:text-[4.0625rem] 2xl:leading-[4.6875rem] text-black-core/[.87] font-roboto-bold v2-header-3-text"
+          class="text-center lg:text-left w-full md:w-[95%] lg:w-[90%] xl:w-[94%] 2xl:w-[81%] mobile-header-1 lg:desk-header-2 xl:desk-header-1"
         >
-          For Over A Decade We Have Helped Businesses Grow Exponentially.
-          <div>
-            — <span class="canopas-gradient-text"> Is Yours Next?</span>
-          </div>
+          For over a decade we have helped businesses grow exponentially.
+          <div>- <span class="primary-color"> Is Yours Next?</span></div>
         </h1>
         <div
-          class="w-[85%] md:w-[65%] pb-[29.5px] md:pb-[47px] v2-normal-text tracking-[0] text-[1rem] leading-[1.375rem] md:text-[1.09375rem] md:leading-[1.53125rem] lg:text-[1.1875rem] lg:leading-7 text-black-core/[0.6] font-inter-regular"
+          class="sub-h1-regular lg:mobile-header-2-regular text-black-core/[0.6] w-full md:w-[75%] lg:w-full 2xl:w-[89%] pt-4 lg:pt-6 text-center lg:text-left"
```

Comment:
as you are unifying the code, then you should start following the standard format for color classes

Reply:
yes i know about that but currently problem is like we have used one class for black color in tailwind config file and used it throughout the website so if i want to create one comman class for it i have to remove it throughout the website so we have decided to do it in last

Reply:
no, i mean use `text-black-core` with `opacity-[0.6]` instead of `text-black-core/[0.6] `

Reply:
its a short form we are using for it isn't it will be more feasible then using long one

Reply:
- yaa I know but as you know what is the importance of standard coding. it's just my suggestion to use in this way.
- we can try our best for future developers otherwise, they will do the same.